src/components/Location.js

```diff
@@ -38,17 +38,22 @@ const Location = (props) => {
 
   return (
     <View>
-      <TouchableOpacity style={styles.icon} onPress={getLocationCoords}>
+      <TouchableOpacity style={styles.iconposition} onPress={getLocationCoords}>
         <Image style={styles.icon} source={require('../img/icongps.png')} />
       </TouchableOpacity>
     </View>
   );
 };
 
 const styles = StyleSheet.create({
+  iconposition:{
+    marginTop: 12,
+    marginHorizontal: 5,
```

Comment:
Przez to obrazek GPSa jest ucięte

Reply:
Bez tego dobrze się wyświetla ikonka w komponencie

Reply:
tak bo element i ikonka byly podpiete do jednego stylu

Reply:
podzieliłem je na osobne